specification/cognitiveservices/OpenAI.Inference/models/completions/chat_completions.tsp

```diff
@@ -303,6 +304,12 @@ model ChatCompletions {
     """)
   choices: ChatChoice[];
 
+  @doc("""
+    The deployment name used for this completions request.
+  """)
+  @clientName("DeploymentName", "csharp")
```

Comment:
I do not believe that this is the deployment name in AOAI.

Reply:
Fixed - removed the clientName and also updated the doc comment to say it's a model.